### PR TITLE
FIX #64 sum up the processing time of all the subqueries and the main

### LINKED ISSUE #64: The `processingTimeMS` should be summed
When we're dealing with disjunctive refinements, it generates multiple queries -> we need to sum the processingTimeMS of each answer to compute the final one.

### PR DESCRIPTION
**Pull request by bobylito**

### CODE REVIEW

File: src/SearchResults/index.js [hunk 1]
```diff
@@ -75,7 +75,13 @@ var SearchResults = function( state, algoliaResponse ) {
    * processing time of the main query
    * @member {number}
    */
-  this.processingTimeMS = mainSubResponse.processingTimeMS;
+  this.processingTimeMS = ( function( responses ) {
```
Comment (vvo) [inline src/SearchResults/index.js]: seems like a good case to use a reduce?

Comment (vvo) [inline src/SearchResults/index.js]: => good opportunity

Comment (pixelastic) [inline src/SearchResults/index.js]: Or using lodash `_.sum(responses, 'processingTimeMS')`

Comment (bobylito) [inline src/SearchResults/index.js]: agreed thanks guys!

**Comment by bobylito** [comment]
I guess we're done here :)

**Pull request merged**